### ISSUE TITLE
Change background color of Weekened placeholder dates. 

### ISSUE THREAD

**Issue by MrCarter31** [kvyatkovskys/KVKCalendar#164]
I am trying to recreate the apple calendar month view on the ipad.   Im able to change the background color of the weekend, but not the weekend placeholder dates.  How would I change the background color of the placeholder dates only on the weekend?

Here is a screenshot down below of what im trying to recreate, notice how the placeholder weekend cell has the same background as the weekend cells.
<img width="1058" alt="Screen Shot 2021-02-28 at 2 45 38 PM" src="https://user-images.githubusercontent.com/50532982/109436644-62f2b400-79d5-11eb-9d3d-67427ab6ef78.png">

This only changes the background of the weekend cells and not the weekend placeholder cells.
`style.month.colorBackgroundWeekendDate = .systemGray2`

**Comment by kvyatkovskys** [comment]
Hello 👋🏼 

If I understand correctly your question. 
Try to change `colorWekeendDate or colorBackgroundWeekendDate `. Let me check in code. 
BTW, you can use `dequeueCell` to customize cells

**Comment by MrCarter31** [comment]
Setting colors for colorWekeenedDate or colorBackgroundWeekendDate did not work.  I also tried using dequeueCell to customize the cells, but im not able to customize the placeholder dates.  Here is a screenshot down below.  I also tried setting showDatesForOtherMonths to true, and those cells still weren't able to be customized.

<img width="1058" alt="Screen Shot 2021-03-01 at 12 16 43 PM" src="https://user-images.githubusercontent.com/50532982/109553714-351a7780-7a88-11eb-92eb-760e6fbbd9be.png">
